app/src/main/AndroidManifest.xml

```diff
@@ -70,8 +70,8 @@
     <activity
       android:name=".settings.profile.ProfileRenameActivity" />
     <activity
-      android:name=".settings.profile.ProfileResetPinActivity"
-      android:screenOrientation="portrait" />
+      android:name=".settings.profile.ProfileResetPinActivity">
+    </activity>
```

Comment:
Refactor this as follows :-

`<activity`
`      android:name=".settings.profile.ProfileResetPinActivity" />`